c/makeotf/makeotf_lib/source/typecomp/subr.c

```diff
@@ -37,11 +39,17 @@
 
    The most challenging part of the process of subroutinization is finding and
    counting the repeated charstrings. This is achieved by first building a suffix
-   DAWG using the concatenation of all the charstrings from all the fonts as
-   input. Subseqeuntly the completed suffix DAWG is traversed in order to count
+   CDAWG using the concatenation of all the charstrings from all the fonts as
+   input. Subseqeuntly the completed suffix CDAWG is traversed in order to count
```

Comment:
Subseqeuntly -> Subsequently

Reply:
I fixed that. I'm ignoring the Codacy warnings for now, as I want to keep this code reasonably close to the `tx` code. If I fix either, I want to fix both. (And yes, ultimately, I'd like to just have one shared library for this, but at least this is a step in that direction.)